new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -98,7 +100,7 @@
    */
   @BeforeAll
   public static void initAll(@Namespaces(3) List<String> namespaces) {
-
+    logger = ThreadSafeLogger.getLogger();
```

Comment:
you can static import ThreadSafeLogger.getLogger then just call getLogger() here like in other It* classes.